tests/unit/test_svs_tiered.cpp

```diff
@@ -2408,8 +2412,8 @@ TYPED_TEST(SVSTieredIndexTestBasic, switchDeleteModes) {
     VecSimParams svs_params = CreateParams(params);
     auto mock_thread_pool = tieredIndexMock();
 
-    auto *tiered_index =
-        this->CreateTieredSVSIndex(svs_params, mock_thread_pool, swap_job_threshold);
+    auto *tiered_index = this->CreateTieredSVSIndex(svs_params, mock_thread_pool,
+                                                    swap_job_threshold, swap_job_threshold);
```

Comment:
Can we rename `swap_job_threshold`? 
Also, is it correct to check
```
    ASSERT_EQ(tiered_index->GetFlatIndex()->indexSize(), 0);
    ASSERT_EQ(tiered_index->GetBackendIndex()->indexLabelCount(), n);
```
in lines 2452-3? How come there are no vectors in the flat buffer if didn't reach the update threshold?

Reply:
Renamed and tests updated.